store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -17,10 +17,14 @@
 package org.apache.rocketmq.store.config;
 
 import java.io.File;
+
 import org.apache.rocketmq.common.annotation.ImportantField;
 import org.apache.rocketmq.store.ConsumeQueue;
 
 public class MessageStoreConfig {
+
+    public static final String MULTI_PATH_SPLITTER = ":";
```

Comment:
YES, it would be confusing. Thank you for the reminding.
A common reserved character would be a nice fit for this splitter.
According to[ Comparison of file Systems](https://en.wikipedia.org/wiki/Comparison_of_file_systems), different files system have different reserved characters. So there is no best character for this splitter,  and I am using ',' as default splitter, user can change it using system property `rocketmq.broker.multiPathSplitter`